src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = (
+  {
+    todo:
+      { completed, title, user },
```

Comment:
I prefer to avoid nested object destructuring.
It's better to follow the approach:
```
export const TodoInfo = ({ todo }) => {
  const { completed, title, user } = todo;
```
It's more readable and more apparent.